src/models/callercalleemodel.h

```diff
@@ -99,24 +99,35 @@ class SymbolCostModelImpl : public HashModel<Data::SymbolCostMap, ModelImpl>
         if (role == Qt::InitialSortOrderRole && column == Cost) {
             return Qt::DescendingOrder;
         }
-        if (role != Qt::DisplayRole) {
+        if (role != Qt::DisplayRole && role != Qt::ToolTipRole) {
             return {};
         }
 
-        switch (column) {
-            case Symbol:
-                return ModelImpl::symbolHeader();
-            case Binary:
-                return QObject::tr("Binary");
-            case Cost:
-                return QObject::tr("Cost");
+        if (role == Qt::DisplayRole) {
+            switch (column) {
+                case Symbol:
+                    return ModelImpl::symbolHeader();
+                case Binary:
+                    return QObject::tr("Binary");
+                case Cost:
+                    return QObject::tr("Cost");
+            }
+        } else if (role == Qt::ToolTipRole) {
+            switch (column) {
+                case Symbol:
+                    return QObject::tr("The function name of the %1. May be empty when debug information is missing.").arg(ModelImpl::symbolHeader());
+                case Binary:
+                    return QObject::tr("The name of the executable the symbol resides in. May be empty when debug information is missing.");
+                case Cost:
+                    return QObject::tr("The number of CPU samples used by this symbol.");
```

Comment:
Does this sound good: "The number of samples attributed to this symbol, both directly and indirectly. This is the symbols inclusive cost."?